packages/amplify_core/lib/src/types/notifications/push/pinpoint_event_source.dart

```diff
@@ -11,8 +11,8 @@
 /// [Journeys](https://docs.aws.amazon.com/pinpoint/latest/userguide/journeys.html)
 /// {@endtemplate}
 enum PinpointEventSource {
```

Comment:
is this type public/exported? if so is this considered a breaking change?

Reply:
I marked the enum as deprecated and created a new private enum with the updated values. thanks for the callout!